Traceback (most recent call last):
  File "C:\Users\behr_mi\Projekte\sumo\repos\sumo\tests\..\tools/devel/debug2shapes.py", line 29, in <module>
    if os.path.isfile(sys.argv[2]):
IndexError: list index out of range
